pages/community/community-projects.md

```diff
@@ -8,8 +8,8 @@ redirect_from: /testimonials/
 ---
 
 {{site.data.alerts.note}}
-Do you use preCICE? We are always happy to hear about interesting projects that use preCICE. <br>
-<a class="button primary" href="community-channels.html">Tell us your story!</a>
+Do you use preCICE? We are always happy to hear about interesting projects that use preCICE. To add your project here, you can directly [create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) to the website repository on GitHub (it is easy also if you don't yet have any GitHub experience!). Simply copy any of the <a class="button primary" href="https://github.com/precice/precice.github.io/tree/master/collections/_testimonials"> previous descriptions</a>, adjust with your data, and <a class="button primary" href="https://github.com/precice/precice.github.io/tree/master/images/testimonials">add a picture</a>.
```

Comment:
@uekerman You are right, Markdown does not render correctly here. Feel free to apply and merge:

```suggestion
Do you use preCICE? We are always happy to hear about interesting projects that use preCICE. To add your project here, you can directly <a href="https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request">create a pull request</a> to the website repository on GitHub (it is easy also if you don't yet have any GitHub experience!). Simply copy any of the <a class="button primary" href="https://github.com/precice/precice.github.io/tree/master/collections/_testimonials"> previous descriptions</a>, adjust with your data, and <a class="button primary" href="https://github.com/precice/precice.github.io/tree/master/images/testimonials">add a picture</a>.
```